.github/workflows/Tests.yaml

```diff
@@ -983,6 +976,7 @@ ${{ matrix.build_type }}-pch-${{ matrix.use_pch || 'ON' }}"
             -D OVERRIDE_ARCH=x86-64 \
             -D BUILD_SHARED_LIBS=ON \
             -D BUILD_PYTHON_BINDINGS=ON \
+            -D BOOTSTRAP_PY_DEPS=ON \
```

Comment:
The `SetupPybind11.cmake` isn't finding the boostrapped pybind11. Maybe we need another hint in cmake?